packages/gatsby-transformer-video/README.md

```diff
@@ -0,0 +1,123 @@
+# gatsby-transformer-video
+
+Convert videos into well supported formats via FFMPEG. Supports generating previews as well.
+
+:warning: Converting videos might take a lot of time. Make sure to have a effective caching mechanism in place. See [caching](#caching)
+
+## Features
+
+- Supports file nodes & Contentful assets
+- Defaults optimized for small files with high visual quality
+- Convert videos to h264 and h265
+- Create animated previews as mp4, animated webp and animated gif.
+- Add watermarks to you videos
+- More?
+
+## Installation
+
+```sh
+npm i gatsby-transformer-video
+```
+
+## Prerequisites
+
+You will need to install `ffmpeg` and `ffprobe` as well.
+
+Go to https://ffmpeg.org/download.html and download either a shared or static version of the library.
+
+Some operating systems may provide these packages via package manager. Some may **not provide** `ffprobe` when installing `ffmpeg`.
+
+You can either:
+
+- Provide the path to ffmpeg && ffprobe via GatsbyJS plugin configuration
+- Set an environment variable -> https://github.com/fluent-ffmpeg/node-fluent-ffmpeg#ffmpeg-and-ffprobe
+- Add the static binaries to any folder in your `$PATH`
+
+### Linux
+
+Linux users might be able to: `sudo apt-get install ffmpeg`
+
+Make sure you have `ffprobe` installed as well.
+
+### OSX
+
+I could not figure out how to use `brew` to install ffprobe as well
```

Comment:
should the `brew install ffmpeg` also install the ffprobe?

i think a link to the official wiki would be the best: https://trac.ffmpeg.org/wiki/CompilationGuide/macOS#ffmpegthroughHomebrew

Reply:
Wonderful, good catch. Tapping the formula makes ffprobe available :)

Reply:
done